hdfs3/tests/test_hdfs3.py

```diff
@@ -27,6 +28,7 @@ def hdfs():
 
     if hdfs.exists('/tmp/test'):
         hdfs.rm('/tmp/test')
+    hdfs.disconnect()
```

Comment:
Why is this required, since `__del__` on the HDFileSystem will call disconnect anyway?